tidb-cloud/set-up-vpc-peering-connections.md

```diff
@@ -223,9 +227,9 @@ Now you have successfully set up the VPC peering connection. Next, [connect to t
     - VPC Network Name
     - VPC CIDR
 
```

Comment:
Change Application Google Cloud Project ID to Google Cloud Project ID
![ededa1a8-6aa8-4a19-a7f2-9aca707e5903](https://github.com/pingcap/docs/assets/35301108/6b13af5f-ddaa-4e43-8928-016248855de0)

Reply:
Done: https://github.com/pingcap/docs/pull/18149/commits/43eb51e46463530a033a410fe1863bbb86cbca8d